pkgs/data/fonts/noto-fonts/default.nix

```
@@ -228,14 +228,14 @@ rec {
   noto-fonts-emoji-blob-bin =
     let
       pname = "noto-fonts-emoji-blob-bin";
-      version = "14.0.1";
+      version = "15.0";
     in
     stdenvNoCC.mkDerivation {
       inherit pname version;
 
       src = fetchurl {
         url = "https://github.com/C1710/blobmoji/releases/download/v${version}/Blobmoji.ttf";
-        hash = "sha256-w9s7uF6E6nomdDmeKB4ATcGB/5A4sTwDvwHT3YGXz8g=";
+        sha256 = "sha256-n5yVk2w9x7UVrMe0Ho6nwu1Z9E/ktjo1UHdHKStoJWc=";
```

It looks like the has for this isnt right (has it changed since this PR was opened?). Was about to open a PR when I saw this, and my hash was different.
Repro:
```
$ nix-prefetch-url --type sha256  https://github.com/C1710/blobmoji/releases/download/v15.0/Blobmoji.ttf
path is '/nix/store/ljcxi6pzqdv71x1r7zpqkavsv23raiy3-Blobmoji.ttf'
1ybqxd3nkx2ychya5jw6k2yvaw17achj4kapxwsrpfina1kxdhyw
$ nix hash to-sri sha256:1ybqxd3nkx2ychya5jw6k2yvaw17achj4kapxwsrpfina1kxdhyw
sha256-3MPWZ1A2ups171dNIiFTJ3C1vZiGy6I8ZF70aUfrePk=
```

```suggestion
        sha256 = "sha256-3MPWZ1A2ups171dNIiFTJ3C1vZiGy6I8ZF70aUfrePk=";
```